grudge/dof_desc.py

```diff
@@ -180,14 +197,35 @@ def __init__(self, domain_tag, quadrature_tag=None):
         else:
             raise ValueError("domain tag not understood: %s" % domain_tag)
 
-        if domain_tag is DTAG_SCALAR and quadrature_tag is not None:
-            raise ValueError("cannot have nontrivial quadrature tag on scalar")
+        # Prints warning if a user passes both a quadrature_tag and
+        # a discretization_tag
+        if (quadrature_tag is not None and discretization_tag is not None):
+            warn("`quadrature_tag` and `discretization_tag` are specified. "
+                 "`quadrature_tag` will be ignored. Use `discretization_tag` "
+                 "instead.",
+                 DeprecationWarning, stacklevel=2)
```

Comment:
This should be an error IMO.

Reply:
Done: https://github.com/inducer/grudge/pull/83/commits/a70b9d6f0b904382bed836331f6bb043cb6ea297